app.py

```diff
@@ -4,9 +4,18 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+
 @app.route('/')
-def index():
-    return 'Hello World!'
+def index(name=None):
+    return render_template('index.html', items=session.get_items())
+
+
+@app.route('/', methods=["POST"])
+def additem():
+    NewToDoItem = request.form["NewToDoItem"]
+    session.add_item(NewToDoItem)
```

Comment:
In Python variable names should be snake_case: https://www.python.org/dev/peps/pep-0008/
```suggestion
    new_todo_item= request.form["NewToDoItem"]
    session.add_item(new_todo_item)
```